src/styles/main.scss

```diff
@@ -1,7 +1,8 @@
+@import '~hds-design-tokens/lib/all.scss';
```

Comment:
I think this won't be needed. I think it duplicates the tokens. I would remove the whole `hds-design-tokens` dependency, because if it is not used, I don't think it is even included in the final bundle.

Reply:
The breakpoints are still used through hds-design-tokens, including them from hds-core's :root we'd get some extra styles that we don't necessarily want to include. Not removing hds-design-tokens in this PR, if a suitable solution is found later and there's time and inclination this can be improved.